source/renderer/app/containers/Root.js

```diff
@@ -27,6 +27,7 @@ export default class Root extends Component<Props> {
 
     const isPageThatDoesntNeedWallets =
       isBlockConsolidationStatusDialog ||
+      isDelegationPage ||
```

Comment:
@yakovkaravelov we need wallets for this page (we will need them to be more precise). Please remove this part.

Reply:
@yakovkaravelov please move this line into the next one so that it reads:
```
((isAdaRedemptionPage || isDelegationPage || isSettingsPage) && hasLoadedWallets && isSynced);
```